### PR TITLE
feat(route): add ollama models

### PR DESCRIPTION
**Pull request by Nick22nd**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/ollama/library
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/ollama/models.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['ollama.com/library'],
+        },
+    ],
+    name: 'ollama models',
```
Comment (TonyRL) [inline lib/routes/ollama/models.ts:22]:
```suggestion
    name: 'Models',
```

File: lib/routes/ollama/models.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    categories: ['programming'],
+    example: '/ollama/library',
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
```
Comment (TonyRL) [inline lib/routes/ollama/models.ts:16]:
```suggestion
```

File: lib/routes/ollama/models.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+            return {
+                title: name.text(),
+                link,
+                description: description.text(),
+            };
+        });
+    return {
+        title: `ollama library`,
+        link: `https://ollama.com/library`,
```
Comment (TonyRL) [inline lib/routes/ollama/models.ts:44]:
```suggestion
        title: 'ollama library',
        link: 'https://ollama.com/library',
```

Comment (Nick22nd) [reply lib/routes/ollama/models.ts]: Thank you for helping review my code.

**Pull request merged**